appveyor.yml

```diff
@@ -11,28 +12,26 @@ environment:
       PYTHON_VERSION: "2.7.x"
       PYTHON_ARCH: "64"
 
-    - PYTHON: "C:\\Python33_64"
-      PYTHON_VERSION: "3.3"
+    - PYTHON: "C:\\Python33-x64"
```

Comment:
@kapilkd13 you can drop ``Python 3.3``, in my opinion. It is of little use and it's slowing down builds